### ISSUE TITLE
Ability to use Documenter{GitHubActions} without GitHubActions CI

### ISSUE THREAD

**Issue by musm** [invenia/PkgTemplates.jl#204]
Is there a way to set things up to make Documenter use GitHubActions, without having GitHubActions set up julia CI?

```
Template(; 
    plugins=[
        Git(ssh=true),
        GitHubActions(),
        Documenter{GitHubActions}(),
    ],
)
```

**Comment by christopher-dG** [comment]
As in you only want docs but not testing with GitHub Actions? If so, then currently no.

**Comment by musm** [comment]
Exactly.

**Comment by musm** [comment]
Another thing that would be nice is to seperate out the two `.yml`  i.e. `CI.yml` and `Documenter.yml`

**Comment by christopher-dG** [comment]
Yeah that would also make it easier to implement this, I have thought about it a little but not come up with something that I really like. I'll look into it more this upcoming week.

**Comment by musm** [comment]
Fantastic thanks again for your work
